compaction/task_manager_module.hh

```diff
@@ -48,23 +50,36 @@ protected:
 
 class major_compaction_task_impl : public compaction_task_impl {
 public:
+    enum class flush_mode {
+        unset,
+        skip,
+        compacted_tables,
+        all_tables
```

Comment:
It is unclear what these latter two options do.

Reply:
I'll add some commentary then.

Reply:
comments added in next version